pages/profiles/[...profile].js

```diff
@@ -571,18 +747,77 @@ Changelog.propTypes = {
   changelog: PropTypes.string.isRequired,
 };
 
-export default function Schema({ schema, changelog, schemaPath }) {
+/**
+ * Displays a tab title with a copy button to copy the URL that opens this tab on page load.
+ */
+function SchemaTab({ id, tabTitle, tabUrl = "", children }) {
+  const { isCopied, initiateCopy } = useCopyAction(tabUrl);
+  const tooltipAttr = useTooltip(`copy-tab-url-${id}`);
+
+  return (
+    <div className="relative">
+      <TabTitle id={id}>{children}</TabTitle>
+      <TooltipRef tooltipAttr={tooltipAttr}>
+        <button
```

Comment:
Why not use CopyButton here?

Reply:
I felt this copy button has to be as unobtrusive as possible, and even the new `sm` option on the copy button was too big and bold to use in these tabs where you normally see nothing unless you hover over the right spot (I don’t actually expect many submitters to use these links — mostly I expect wranglers to use them; I could be wrong).

So I followed the same principle I have for using the `form-elements/button`  module — use it if you can. Do something else if you can’t.

Reply:
If only wranglers use it then it should not be a problem. I just had a hard time to find those copy buttons.

Reply:
Thanks. We’ll see what happens in QA.